### ISSUE TITLE
Add new class to the labels and finetune with COCO weights

### ISSUE THREAD

**Issue by AriChow** [dbolya/yolact#211]
Hi,
Is there a way to add a new label to the set of labels in COCO (e.g. house) and finetune on the pretrained weights (e.g. yolact_base_54_800000.pth) instead of training from scratch?

**Comment by dbolya** [comment]
Yeah, check #36 for how to set up fine tuning. Then all you need to do is add that class to your dataset and increase the number of classes in the config (just follow exactly the same steps as for any other custom dataset, taking care that label_map is correct since COCO labels IDs are weird).

What'll happen when you resume from the pretrained weights is that torch will find the size of the weights saved for the final linear layer doesn't match the current size (since you increased the number of classes), so only that layer will remain uninitialized and thus you can keep the rest of the network from the pretrained weights.

**Comment by AriChow** [comment]
Thanks @dbolya !
